sensing/gnss_poser/include/gnss_poser/gnss_stat.hpp

```diff
@@ -16,12 +16,7 @@
 
 namespace gnss_poser
 {
-enum class CoordinateSystem {
-  UTM = 0,
-  MGRS = 1,
-  PLANE = 2,
-  LOCAL_CARTESIAN = 3,
-};
+enum class CoordinateSystem { UTM = 0, MGRS = 1, PLANE = 2, LOCAL_CARTESIAN = 3, UTM_LOCAL = 4 };
```

Comment:
You can change options as LOCAL_CARTESİAN_WGS84 and LOCAL_CARTESİAN_UTM

```suggestion
enum class CoordinateSystem { UTM = 0, MGRS = 1, PLANE = 2, LOCAL_CARTESIAN_WGS84 = 3, LOCAL_CARTESIAN_UTM = 4 };
```